python/faery/events_stream.py

```diff
@@ -615,3 +657,209 @@ class FiniteRegularEventsFilter(
     stream.FiniteRegularFilter[numpy.ndarray],
 ):
     pass
+
+
+@dataclasses.dataclass
+class PacketState:
```

Comment:
This seems like a good idea. Can I ask you to add one or two lines of documentation, just to make it clear what this does. I assume it's what helps you keep track of stream states, like where in a "chunked" event stream you are?

Reply:
Will do!